source/_integrations/webostv.markdown

```diff
@@ -24,123 +24,43 @@ There is currently support for the following device types within Home Assistant:
-
-Pairing information will be saved to a configuration file `webostv.conf` in the Home Assistant configuration directory. This process is IP address-sensitive, in case the IP address of your TV would change in future.
-
-## Configuration
-
-To add a TV to your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-webostv:
-```
-
-{% configuration %}
-host:
-  description: "The IP of the LG webOS Smart TV, e.g., `192.168.0.10`."
-  required: true
-  type: string
-name:
-  description: The name you would like to give to the LG webOS Smart TV.
-  required: false
-  type: string
-turn_on_action:
-  description: Defines an [action](/docs/automation/action/) to turn the TV on.
-  required: false
-  type: string
-customize:
-  description: List of options to customize.
-  required: false
-  type: map
-  keys:
-    sources:
-      description: List of hardware and webOS App inputs.
-      required: false
-      type: list
-{% endconfiguration %}
-
-### Full configuration example
-
-A full configuration example will look like the sample below:
-
-```yaml
-# Example configuration.yaml entry
-webostv:
-  host: 192.168.0.10
-  name: Living Room TV
-  turn_on_action:
-    service: persistent_notification.create
-    data:
-      message: "Turn on action"
-  customize:
-    sources:
-      - livetv
-      - youtube
-      - makotv
-      - netflix
-
-media_player:
-
-notify:
-```
-
-Avoid using `[ ]` in the `name:` of your device.
-
-### Using multiple TVs
-
-It is also possible to use multiple TVs with this integration.
-
-```yaml
-# Example configuration.yaml entry with multiple TVs
-webostv:
-  - name: Living Room TV
-    host: 192.168.1.100
-  - name: Bedroom TV
-    host: 192.168.1.101
-```
-
-Please note, the above provides a minimal example, however, all options are
-available for each individual TV.
+{% include integrations/config_flow.md %}
 
 ## Turn on action
 
 Home Assistant is able to turn on a LG webOS Smart TV if you specify an action, like HDMI-CEC or WakeOnLan.
 
-Common for webOS 3.0 and higher would be to use WakeOnLan feature. To use this feature your TV should be connected to your network via Ethernet rather than Wireless and you should enable *LG Connect Apps* feature in *Network* settings of the TV [instructions](https://www.lg.com/uk/support/product-help/CT00008334-1437131798537-others) (or *Mobile App* in *General* settings for older models) (*may vary by version).
+Common for webOS 3.0 and higher would be to use WakeOnLan feature. To use this feature your TV should be connected to your network via Ethernet rather than Wireless and you should enable *LG Connect Apps* feature in *Network* settings of the TV (or *Mobile App* in *General* settings for older models) (*may vary by version).
```

Comment:
Suggested edit: "...enable the LG Connect Apps feature..." (add 'the')

Reply:
Added, thanks